example/android/app/src/main/java/com/klaviyoreactnativesdkexample/MainActivity.kt

```diff
@@ -17,4 +33,139 @@ class MainActivity : ReactActivity() {
    * which allows you to enable New Architecture with a single boolean flags [fabricEnabled]
    */
   override fun createReactActivityDelegate(): ReactActivityDelegate = DefaultReactActivityDelegate(this, mainComponentName, fabricEnabled)
+
+  /**
+   * Launches a permission request, and receives the result in the callback below
```

Comment:
Add comment that this is optional for native, and can be done from RN as alternative?